http/prewiew.go

```diff
@@ -0,0 +1,94 @@
+package http
+
+import (
+	"image"
+	"net/http"
+	"net/url"
+
+	"github.com/disintegration/imaging"
+
+	"github.com/filebrowser/filebrowser/v2/files"
+)
+
+const sizeThumb = "thumb"
+const sizeBig = "big"
```

Comment:
Please group constants
```suggestion
const (
	sizeThumb = "thumb"
	sizeBig = "big"
)
```